### DISCUSSION TITLE
Podcasts? How to add

### DISCUSSION THREAD

**Discussion by olinorwell** [advplyr/audiobookshelf#557] [Q&A]
Now I'm risking looking like a complete fool here, but I'll ask the question anyway.

I use Audiobookshelf and it's great, I have my audiobooks in there and they all appear beautifully on the shelf. I couldn't be happier.

I then keep reading about the new podcast implementation - and people are asking questions and suggesting improvements.

My question is - how on earth do I add podcasts?? I don't see a sidebar item that says "Podcasts". I can't see any mention of an option to enable them in settings.

I used the Ubuntu installation documentation (so no Docker), and the installation was a breeze.

Am I missing something really obvious?? I don't see any mention of podcasts anywhere in the software, I am using 2.0.8 or 2.0.9 I think (it updated yesterday).

I've looked for over an hour, perhaps it's me being stupid but in that case for people like me a extra paragraph in the documentation explaining the podcast feature would be extremely helpful.

So yeah, how do I add podcasts to Audiobookshelf?

**Comment by advplyr** [comment]
You need to create a new library with the library type of podcast.

The docs still need to get updated for podcasts.
